package.json

```diff
@@ -7,12 +7,25 @@
   "author": "Akshay S Dinesh <asdofindia@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build": "jspm bundle src/ascii-unicode-converter.js dist/bundle.js --skip-source-maps"
+    "run-server": "node_modules/http-server/bin/http-server --cors -p 9000",
+    "postinstall": "cp node_modules/siimple/dist/*.min.css src/assets/ && cp node_modules/siimple-colors/dist/*.min.css src/assets/",
+    "build:css": "cp -R src/assets/ dist/",
```

Comment:
Why cp to src postinstall and then to dist during build?
Why not directly copy from the real source folder (which's the node_modules) to dist folder during build?